src/flapjack/transport_test.go

```diff
@@ -12,4 +12,28 @@ func TestDialFails(t *testing.T) {
 	}
 }
 
-// TODO(auxesis): add test for sending and receiving Events
+func TestSendSucceeds(t *testing.T) {
+  transport, _ := Dial("localhost:6379", 9)
```

Comment:
Depends what we want to test. If we want to test that it's calling `event.IsValid()` and marshalling the data for sending, then mock, if we want to test that and it passes through redigo to Redis OK, then don't.

I'd say not, and assume the other parts are already well tested, in the interests of reducing complexity... but it's a question with no right or wrong answer *per se*.